staging/src/k8s.io/apimachinery/pkg/util/validation/validation_test.go

```diff
@@ -154,6 +154,30 @@ func TestIsValidPortNum(t *testing.T) {
 	}
 }
 
+func TestIsInRange(t *testing.T) {
+	goodValues := []struct {
+		value int
+		min   int
+		max   int
+	}{{1, 0, 10}, {5, 5, 20}, {25, 10, 25}}
+	for _, val := range goodValues {
+		if msgs := IsInRange(val.value, val.min, val.max); len(msgs) != 0 {
+			t.Errorf("expected true for %d, got %v", val, msgs)
```

Comment:
again, I don't like the use of the boolean words `true` and `false` in these errors, because IsInRange doesn't deal in booleans
suggest: `t.Errorf("expected no errors for %#v, but got %v", val, msgs)`
`%d` isn't correct because val is not an integer.
similarly for `badValues`